Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/storage/EventRegistrationStorageModule.kt

```diff
@@ -0,0 +1,16 @@
+package de.rki.coronawarnapp.eventregistration.storage
+
+import dagger.Module
+import dagger.Provides
+import de.rki.coronawarnapp.eventregistration.storage.repo.DefaultHostedEventRepository
+import de.rki.coronawarnapp.eventregistration.storage.repo.HostedEventRepository
+import javax.inject.Singleton
+
+@Module
+class EventRegistrationStorageModule {
+
+    @Singleton
+    @Provides
+    fun hostedEventRepository(defaultHostedEventRepository: DefaultHostedEventRepository): HostedEventRepository =
+        defaultHostedEventRepository
+}
```

Comment:
We have a module for EventRegistration already , why new one ?

Reply:
True. I made a new commit where I use the existing one.